.github/workflows/lint.yml

```
@@ -19,4 +19,8 @@ jobs:
 
     - name: "Check for warnings emitted by xml2rfc"
       run: |
-        bash -c -o pipefail 'make |& (! grep -E "Warning|Error")'
+        bash -c -o pipefail <<EOF
+make |& \
+  (! grep -E "Warning|Error)") | \
+  (! grep -v 'Warning: Found SVG with width or height specified, which will make the artwork not scale.  Specify a viewBox only to let the artwork scale.')
```

We need to filter this message out first, since we're using the error code of the sub-shell inside the parentheses
```suggestion
          grep -v 'Warning: Found SVG with width or height specified, which will make the artwork not scale.  Specify a viewBox only to let the artwork scale.' | \
          (! grep -E "Warning|Error)")
```

---

Hmm... I took your suggestion but I'm not sure it had the intended effect. Here's what I've gotten working on the command line:
```bash
make |& (! grep -E "Warning|Error") | (! grep -v 'Warning: Found SVG with width or height specified, which will make the artwork not scale.  Specify a viewBox only to let the artwork scale.')
```

---

I'm trying to figure out how to plumb this into GH actions. I'll have to resume tomorrow however.